.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,51 @@
+## Related Issue
+
+<!-- Link related issue -->
+
+## Description
+
+<!-- Describe what this PR changes. -->
+
+## Review Procedure
+
+<!-- Explain how to review this PR. -->
+
+## Related PR(optional)
+
+<!-- Link related PR -->
+
+## Remarks(optional)
+
+<!-- Write remarks as you like if you need them. -->
+
+## Pre-Review Checklist for the PR Author
+
+**PR Author should check the checkboxes below when creating the PR.**
+
+- [ ] Check documentation
+- [ ] Check sample files
```

Comment:
It might be nice if we say that this is only for people who are adding a new package.
e.g. 
```
If you are adding new package: 
- [ ] Documentation with description of the package is available
- [ ] A sample launch file and parameter file are available if the package contains executable nodes 
```

Reply:
46b5813 added